src/main/scala/tilelink/Bundles.scala

```diff
@@ -15,34 +18,34 @@ abstract class TLBundleBase(params: TLBundleParameters) extends GenericParameter
-  def AcquireBlock   = UInt(6) //     .                        => Grant[Data]
-  def AcquirePerm    = UInt(7) //     .                        => Grant[Data]
-  def Probe          = UInt(6) //          .                   => ProbeAck[Data]
-  def AccessAck      = UInt(0) //               .    .
-  def AccessAckData  = UInt(1) //               .    .
-  def HintAck        = UInt(2) //               .    .
-  def ProbeAck       = UInt(4) //               .
-  def ProbeAckData   = UInt(5) //               .
-  def Release        = UInt(6) //               .              => ReleaseAck
-  def ReleaseData    = UInt(7) //               .              => ReleaseAck
-  def Grant          = UInt(4) //                    .         => GrantAck
-  def GrantData      = UInt(5) //                    .         => GrantAck
-  def ReleaseAck     = UInt(6) //                    .
-  def GrantAck       = UInt(0) //                         .
+  def PutFullData    = 0.U     //     .    .                   => AccessAck
+  def PutPartialData = 1.U     //     .    .                   => AccessAck
+  def ArithmeticData = 2.U     //     .    .                   => AccessAckData
+  def LogicalData    = 3.U     //     .    .                   => AccessAckData
+  def Get            = 4.U     //     .    .                   => AccessAckData
+  def Hint           = 5.U     //     .    .                   => HintAck
+  def AcquireBlock   = 6.U     //     .                        => Grant[Data]
+  def AcquirePerm    = 7.U     //     .                        => Grant[Data]
+  def Probe          = 6.U     //          .                   => ProbeAck[Data]
+  def AccessAck      = 0.U     //               .    .
+  def AccessAckData  = 1.U     //               .    .
+  def HintAck        = 2.U     //               .    .
+  def ProbeAck       = 4.U     //               .
+  def ProbeAckData   = 5.U     //               .
+  def Release        = 6.U     //               .              => ReleaseAck
+  def ReleaseData    = 7.U     //               .              => ReleaseAck
+  def Grant          = 4.U     //                    .         => GrantAck
+  def GrantData      = 5.U     //                    .         => GrantAck
+  def ReleaseAck     = 6.U     //                    .
+  def GrantAck       = 0.U     //                         .
 
   def isA(x: UInt) = x <= AcquirePerm
   def isB(x: UInt) = x <= Probe
   def isC(x: UInt) = x <= ReleaseData
   def isD(x: UInt) = x <= ReleaseAck
 
-  def adResponse = Vec(AccessAck, AccessAck, AccessAckData, AccessAckData, AccessAckData, HintAck, Grant, Grant)
-  def bcResponse = Vec(AccessAck, AccessAck, AccessAckData, AccessAckData, AccessAckData, HintAck, ProbeAck, ProbeAck)
+  def adResponse = VecInit(AccessAck, AccessAck, AccessAckData, AccessAckData, AccessAckData, HintAck, Grant, Grant)
+  def bcResponse = VecInit(AccessAck, AccessAck, AccessAckData, AccessAckData, AccessAckData, HintAck, ProbeAck, ProbeAck)
```

Comment:
Does this break if you make it a `Seq`?

Reply:
@sequencer Can you weigh in? I'm actually not sure what the semantics are